resources/analysisTools/indelCallingWorkflow/confidenceAnnotation_Indels.py

```diff
@@ -510,6 +510,9 @@ def main(args):
         if dbsnp_id is not None and dbsnp_pos is not None:
             entries[2] = dbsnp_id + "_" + dbsnp_pos
 
+        ### Change the columns to make sure control is always in column 9 and tumor in column 10 (0 based)
+        entries[9], entries[10] = entries[header_indices["CONTROL_COL"]], entries[header_indices["TUMOR_COL"]]
```

Comment:
What is the effect of that `header_indices[...]` actual values are set at the top of the script (line 134)?

Reply:
That will be an excellent test case here, where `args.controlColName` is not present in the original column names and with the swapped tumor and control columns. I could test this scenario and see what happens with this version.

Reply:
If the sample name got via BAM file's' `SM` tag is not present in the VCF header, and there is a switch in the control and tumor columns, then the list of high confidence somatic variants is wrong. So far, alignment workflow properly sets them in the BAM file, so no big issue. 

Assigning column number as in line 134 is a destructive behaviour, so the program should exit if they do not find the sample names in the VCF header. 

@vinjana Will this be OK, killing the workflow if the sample name from SM tag is not in the VCF header? Will this be also OK with the OTP settings? Or do you have any alternative ideas?